### ISSUE TITLE
ReferenceError: Can't find variable: android

### ISSUE THREAD

**Issue by tralves** [nstudio/nativescript-audio#167]
Latest commit (254e0719302566b718c578b6933774beac2cffed) broke the plugin on iOS.

Here's the relevant part of the log:

```
*** Terminating app due to uncaught exception 'NativeScript encountered a fatal error: ReferenceError: Can't find variable: android
at
file: node_modules/nativescript-audio/options.js:15:0
at ../node_modules/nativescript-audio/options.js(file: node_modules/nativescript-audio/options.js:19:1)
at __webpack_require__(file: app/webpack/bootstrap:74:0)
at ../node_modules/nativescript-audio/audio.js(file: node_modules/nativescript-audio/audio.ios.js:7:21)
```

 👀@danieldspx.

Thanks

**Comment by tralves** [comment]
I can see someone already fixed it in #166.

**Comment by bradmartin** [comment]
Published the fix
